packages/vscode_simple_ext/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const LOGGING_LEVEL_PROP = "vscode-simple-ext.loggingLevel";
```

Comment:
I think these constants should be defined in `logger.ts`. They are related to logging and they are not used anywhere else.

Reply:
**This is intentional.**
I won't be able to (easily) import logger.ts in unit tests without mocks / proxyquire because of the dependency to  vscode. 

Nor do I want to add the complexity of those mocks or DI workaround in the context of this example.

Reply:
Okay